documentation/docs/tutorials/openfga.mdx

```diff
@@ -0,0 +1,105 @@
+# OpenFGA and OPAL
+
+[OpenFGA](https://openfga.dev) is an open-source authorization engine implementing the Zanzibar authorization model created by Google. OPAL provides realtime synchronization and management capabilities for OpenFGA, allowing you to keep your authorization data and relationships up-to-date across distributed OpenFGA instances.
+
+## Quick Start
+
+OPAL can run OpenFGA as a policy engine instead of OPA. To launch an example configuration with Docker-Compose:
+
+```bash
+git clone https://github.com/permitio/opal.git
+cd opal
+docker-compose -f docker/docker-compose-example-openfga.yml up -d
+```
+
+## Interacting with OpenFGA
+
+The OpenFGA API will be available at [http://localhost:8080](http://localhost:8080). Here are some key endpoints and operations:
```

Comment:
Maybe I'm missing something, but what happened if you send data to OpenFGA? In OPA examples, we are not showing that. I think it's better to just refer to the OpenFGA OpenAPI spec https://openfga.dev/api/service

The interacting section should show and describe the APIs that are available from the OPAL side